Plugins/NativeEngine/Source/VertexBuffer.cpp

```diff
@@ -142,11 +147,17 @@ namespace Babylon
     {
         if (m_dynamic)
         {
-            encoder->setVertexBuffer(stream, m_dynamicHandle, startVertex, numVertices, layoutHandle);
+            if (bgfx::isValid(m_dynamicHandle))
```

Comment:
You can do this check once at the top. Yes, it's a union, but they are both identical layout, so it should be fine. We already do this the CreateHandle function.

Reply:
Updated on latest commit.